courses/sync_external_courses/external_course_sync_api_test.py

```diff
@@ -513,6 +513,36 @@ def test_create_or_update_external_course_run(
         assert getattr(course_runs[0], attr_name) == expected_value
 
 
+@pytest.mark.parametrize(
+    "external_course_data",
+    [{"platform": EMERITUS_PLATFORM_NAME}, {"platform": GLOBAL_ALUMNI_PLATFORM_NAME}],
+    indirect=True,
+)
+@pytest.mark.parametrize(
+    ("enrollment_end_date", "expected_enrollment_end"),
+    [
+        ("2099-10-15", "2099-10-15"),
+        (None, "2099-10-07"),  # start_date + 7 days
+    ],
+)
+@pytest.mark.django_db
+def test_enrollment_end_logic(
```

Comment:
This does not test the case when enrollment end date key is not present in the API data.